clients/admin-ui/src/features/datastore-connections/system_portal_config/forms/ConnectorParametersForm.tsx

```diff
@@ -258,10 +271,39 @@ const ConnectorParametersForm: React.FC<ConnectorParametersFormProps> = ({
         }
       });
     }
-    onSaveClick(updatedValues, actions);
+    return updatedValues;
+  };
+
+  const handleSubmit = (values: any, actions: any) => {
+    // convert each property value of type FidesopsDatasetReference
+    // from a dot delimited string to a FidesopsDatasetReference
+    const processedValues = preprocessValues(values);
+    onSaveClick(processedValues, actions);
+  };
+
+  const handleAuthorizeConnectionClick = async (
+    values: ConnectionConfigFormValues,
+    // @ts-ignore
```

Comment:
Can you explain why the `@ts-ignore` is needed in a comment?

Reply:
I was following the convention we were using in other places but it looks like the correct approach is to do something like `FormikProps<ConnectionConfigFormValues>` and specify the actual form value type. I made this update in a few places to remove the need for `@ts-ignore`